ocvbot/behavior.py

```
@@ -22,6 +22,100 @@
 
 # TODO: Move login and world-switcher functions to login_menu.py.
 # TODO: Add switch_worlds_logged_in()
+def switch_worlds_logged_in(world: str, attempts=5) -> bool:
+    # Number of worlds visible at a time
+    WORLDS_PER_PAGE = 12
+
+    # Timeout after clicking the world
+    TIMEOUT = 20
+
+    # Region where current world is displayed
+    CURRENT_WORLD_TEXT = (vis.INV[0], vis.INV[1], 165, 20)
+
+    # Get world's row and col
+    world_info = start.worlds[world]
+    column = world_info["column"]
+    row = world_info["row"]
+
+    # Position of world on world list
+    index = (((WORLDS_PER_PAGE * column) - WORLDS_PER_PAGE) * 2) + row
+
+    # Scroll bar position
+    scroll_bar_x = vis.CLIENT[0] + 722
+    scroll_bar_y = vis.CLIENT[1] + 256
+
+    # Calculate position of world on scroll bar
+    scroll_bar_y += int(9 + (index - WORLDS_PER_PAGE) * (75 / 112))
+
+    banking.close_bank()
+    open_side_stone("logout")
+
+    # Check if world switcher is open
+    try:
+        open = vis.Vision(
+            region=vis.INV,
+            needle="./needles/side-stones/logout/world-switcher-loaded.png",
+        ).find_needle()
+    except:
+        open = False
+
+    # Open world switcher
+    if not open:
+        try:
+            vis.Vision(
+                region=vis.INV_BOTTOM, needle="./needles/login-menu/world-switcher.png"
+            ).click_needle()
+        except:
+            log.info("Failed to click world switcher!")
+            return False
```

Raise an exception if an error occurs rather than returning `False`.